src/pamiq_core/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import interfaces
+from .interfaces import InferenceModel, TrainingModel
```

Comment:
```suggestion
from .interfaces import InferenceModel, TrainingModel

__all__ = ["InferenceModel", "TrainingModel"]
```

`from model import *`でワイルドカードインポートできるようにしておくと良いです。通常のコードでは使いませんが、簡易的に書いてみるときに楽なので良いです。